hist/hist/src/TH1.cxx

```diff
@@ -9507,7 +9509,7 @@ void TH1C::AddBinContent(Int_t bin)
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Increment bin content by w.
+/// Increment bin content by Int_t(w).
```

Comment:
Here I would like this type of coerced casting to be more explicitly described.
```suggestion
/// Increment bin content by w.
/// \warning The value of w is cast to `Int_t` before being added
```

And similarly for all other functions that do this